packages/augur-sdk/src/utils.ts

```diff
@@ -165,3 +166,52 @@ export function calculatePayoutNumeratorsArray(
   }
   return payoutNumerators;
 }
+
+export function getOutcomeDescriptionFromOutcome(
```

Comment:
Markets.`getMarketOutcomes` needs to use this method. 

make constants for `Yes`, `No` and `Invalid`

Reply:
I agree but getting it integrated without breaking that getter is out of scope. I'll add a TODO so we get a ticket to do that refactor.

Constants added! I put them in an enum.